lib/sisimai/bite/email/sendmail.rb

```diff
@@ -0,0 +1,290 @@
+module Sisimai::Bite::Email
+  module Sendmail
+    # Sisimai::Bite::Email::Sendmail parses a bounce email which created by
+    # v8 Sendmail. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Sendmail.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AMail Delivery Subsystem/,
+        :subject => %r/(?:see transcript for details\z|\AWarning: )/,
+      }
+      # Error text regular expressions which defined in sendmail/savemail.c
+      #   savemail.c:1040|if (printheader && !putline("   ----- Transcript of session follows -----\n",
+      #   savemail.c:1041|          mci))
+      #   savemail.c:1042|  goto writeerr;
+      #
+      Re1 = {
+        :begin   => %r/\A[ \t]+[-]+ Transcript of session follows [-]+\z/,
+        :error   => %r/\A[.]+ while talking to .+[:]\z/,
+        :rfc822  => %r{\AContent-Type:[ ]*(?:message/rfc822|text/rfc822-headers)\z},
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'V8Sendmail: /usr/sbin/sendmail'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->